reframe/core/schedulers/pbs.py

```diff
@@ -31,11 +31,15 @@
 class PbsJobScheduler(sched.JobScheduler):
     def __init__(self):
         self._prefix = '#PBS'
-        self._time_finished = None
+        self._completion_time = None
 
         # Optional part of the job id refering to the PBS server
         self._pbs_server = None
 
+    @property
+    def completion_time(self):
+        return self._completion_time
```

Comment:
The PBS scheduler should return `None`. It doesn't do anything more accurate that the `Job` does. So again, you can leave untouched the rest of its implementation.